components/aws-authservice/README.md

```diff
@@ -0,0 +1,35 @@
+# AWS AuthService
+
+## Overview
+AWS AuthService is an HTTP Server that handles the logging out of an Authenticated user who was connected to Kubeflow using AWS Cognito and Amazon ALB.
+
+## Design
+An HTTP Server that listens for a users logout request that then follows the two steps necessary to logout an Authenticated Cognito + ALB user. These being expiring any ALB Cookies and then hitting the Cognito Logout Endpoint. Official [Documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#authentication-logout) lists these steps as required for secure logout.
+
+## Manifests
+To install AWS AuthService apply them to your EKS Cluster. The manifests can be found in [awsconfigs](../../awsconfigs/common/aws-authservice/base/).
+
+```
+kubectl apply -k ../../awsconfigs/common/aws-authservice/base/
+```
+
+## Options
```

Comment:
makes it sound like this step is optional. I think you can combine this in the manifess

I think these sections(manifests, options, build) are little confusing. What you might want to do it create a Build and test section and walk through what are the steps you need to do incase you are making a change, manifests and configurable parameters can come in that section